pkg/ui/src/views/app/components/layoutSidebar/index.tsx

```diff
@@ -78,11 +78,15 @@ function LoginIndicator({ loginState, handleLogout }: LoginIndicatorProps) {
   }
 
   return (
-    <div className="login-indicator">
-        Logged in as {user}
-        <br />
-        <button onClick={handleLogout}>Logout</button>
-    </div>
+    <li className="login-indicator">
+        <div
+          className="login-indicator__initial"
+          title={`Signed in as ${user}`}
+        >
+          { user[0] }
+        </div>
+        <Link to="/logout" onClick={handleLogout}>Sign Out</Link>
```

Comment:
was gonna suggest that you make the whole thing clickable, but maybe the log out button should be pretty small so you don't click it accidentally.

Reply:
Yep.  Maybe one day we'll have a user menu of some kind?!